core/file_test.go

```diff
@@ -69,6 +70,62 @@ func TestLoadTomlFile_FileExitsShouldPass(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestSaveTomlFile(t *testing.T) {
+	t.Parallel()
+
+	t.Run("empty filename, should fail", func(t *testing.T) {
+		t.Parallel()
+
+		cfg := &TestStruct{}
+
+		fileName := ""
+
+		err := core.SaveTomlFile(cfg, fileName)
+		require.Error(t, err)
+	})
+
+	t.Run("invalid path, should fail", func(t *testing.T) {
+		t.Parallel()
+
+		cfg := &TestStruct{}
+
+		fileName := "invalid_path" + "/testFile1"
+
+		err := core.SaveTomlFile(cfg, fileName)
+		require.Error(t, err)
+	})
+
+	t.Run("should work with valid dir", func(t *testing.T) {
+		t.Parallel()
+
+		cfg := &TestStruct{}
+
+		dir := t.TempDir()
+		fileName := dir + "/testFile1"
+
+		err := core.SaveTomlFile(cfg, fileName)
+		if _, errF := os.Stat(fileName); errF == nil {
+			_ = os.Remove(fileName)
```

Comment:
this is not required as when the test will end, the whole t.TempDir() will be deleted

Reply:
removed